src/gas/GasFeeController.ts

```diff
@@ -218,7 +225,7 @@ export class GasFeeController extends BaseController<typeof name, GasFeeState> {
       never,
       never
     >;
-    state?: Partial<GasFeeState>;
+    state?: GasFeeState;
```

Comment:
State is now very much required to have all pieces to write it, but that doesn't always have to be true. if in the future one of the state keys is optional in all shapes, we can add it to the shapes with the ?: syntax so that partial spreads are supported, but this would still be GasFeeState versus Partial<GasFeeState>